gh-135993: Fix IPv6 bug in `set_ok_port` and `return_ok_port` (#136076)
python/cpython: Misc/NEWS.d/next/Library/2025-06-28-14-10-07.gh-issue-135993.Gmyux9.rst

> :mod:`http.cookiejar` Fix the bug happens when trying to prase ports of a IPv6 address in :class:`~http.cookiejar.DefaultCookiePolicy`

Reviewer: ```suggestion
:mod:`http.cookiejar`: Fix a bug that occurs when :class:`~http.cookiejar.DefaultCookiePolicy`
attempts to parse the port of an IPv6 address.
```
Little grammar/rewording suggestion

Author: LGTM
